src/Workspaces/CoreTest/ReferencedSymbolTests.cs

```diff
@@ -1,288 +0,0 @@
-﻿// Licensed to the .NET Foundation under one or more agreements.
-// The .NET Foundation licenses this file to you under the MIT license.
-// See the LICENSE file in the project root for more information.
-
-#nullable disable
-
-using System;
-using System.Collections.Generic;
-using System.Collections.Immutable;
-using System.Globalization;
-using System.Threading;
-using Microsoft.CodeAnalysis.FindSymbols;
-using Microsoft.CodeAnalysis.Test.Utilities;
-using Roslyn.Test.Utilities;
-using Xunit;
-
-namespace Microsoft.CodeAnalysis.UnitTests
-{
-    public class ReferencedSymbolTests : TestBase
-    {
-        [Fact, Trait(Traits.Feature, Traits.Features.FindReferences)]
-        public void DebuggerDisplay_OneReference()
-        {
-            var referencedSymbol = CreateReferencedSymbol("Goo", 1);
-
-            Assert.Equal("Goo, 1 ref", referencedSymbol.GetTestAccessor().GetDebuggerDisplay());
```

Comment:
these testswere useless.  1. testing debugger display is not necessary.  2. it was doing this bizarre thing about ensuring that hte debugger display only accesses the name of a symbol.  all entirely unnecessary.